examples/multisig-example/src/main/kotlin/network/finschia/sdk/example/CreateValidator.kt

```diff
@@ -0,0 +1,262 @@
+/**
+ * Copyright 2023 Finschia Foundation
+ *
+ * LINE Corporation licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+package network.finschia.sdk.example
+
+import network.finschia.sdk.legacymultisig.*
+import com.google.protobuf.ByteString
+import com.google.protobuf.kotlin.toByteString
+import cosmos.base.v1beta1.CoinOuterClass
+import cosmos.staking.v1beta1.Tx
+import io.grpc.ManagedChannelBuilder
+import kotlinx.serialization.json.*
+import network.finschia.sdk.account.HDWallet
+import network.finschia.sdk.account.KeyWallet
+import network.finschia.sdk.base.protoDecimalToJson
+import org.bouncycastle.jcajce.provider.digest.SHA256
+import java.util.*
+
+class MultiSigMsgCreateValidator {
+    companion object {
+        fun createMsgCreateValidator(
+            deliAddr: String,
+            opAddr: String,
+            amount: CoinOuterClass.Coin,
+            pubKey: String,
+        ): Tx.MsgCreateValidator {
+            val pubKeyBytes = Base64.getDecoder().decode(pubKey).toByteString()
+            val header = ByteArray(2)
+            header[0] = 0x0a
+            header[1] = pubKeyBytes.size().toByte()
```

Comment:
Please add why this value are used.

Reply:
I extracted this code block to the extension function to express purpose.